testutils/rand/rand.go

```diff
@@ -213,7 +213,7 @@ type StringGen struct {
 
 // Strings returns a random string generator that produces strings from the default alphabet of random length in the given limits (upper limit exclusive)
 func Strings(minLength int, shorterThan int) StringGen {
-	alphabet := []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_.:")
+	alphabet := []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-.:")
```

Comment:
Valid strings don't include `_` so I'm removing it here. Should add invalid string generation helpers in a separate PR (underscores, some unnormalized runes etc.)